### PR TITLE
completed ticket 6

### PR DESCRIPTION
**Pull request by zahraakarim**

### CODE REVIEW

File: news.model.js [hunk 1]
```diff
@@ -1,29 +1,42 @@
-const db = require('./db/connection')
-const response = require('./app');
-const comments = require('./db/data/test-data/comments');
-const articles = require('./db/data/test-data/articles');
+const db = require("./db/connection");
+const response = require("./app");
+const comments = require("./db/data/test-data/comments");
+const articles = require("./db/data/test-data/articles");
```
Comment (daviidbartlett) [inline news.model.js:5]: `response`, `comments`, `articles`, why are these here? They're not being used.

File: utils.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const db = require("./db/connection");
+const app = require("./app");
```
Comment (daviidbartlett) [inline utils.js:2]: why is `app` here?

**Comment by daviidbartlett** [comment]
please add a description to you pull requests which describe what functionality you've added to your app. This allows the person who's taking the reviewer know what they're expecting to see.

File: utils.js (hunk 2 above)
Comment (zahraakarim) [reply utils.js:2]: Thought I needed it for my article_id

File: news.model.js (hunk 1 above)
Comment (zahraakarim) [reply news.model.js:5]: Is response not being used? My bad with the other 2, forgot to remove when figuring stuff out but not being used so will remove on my next branch c:
